Orange/widgets/classify/owsaveclassifier.py

```diff
@@ -22,7 +22,9 @@ class OWSaveClassifier(widget.OWWidget):
     #: A list of recent filenames.
     history = Setting([])
 
-    FILTER = "Pickle files (*.pickle *.pck)\nAll files (*.*)"
+
+    SUPPORTED_EXTENSIONS = ('.pkcls', '.pickle', '.pkl')
```

Comment:
We're beginning to have too many extensions. Can we decide for just one? I like pkcls. Same for data, where we would use pkdata.